src/kaleidoscope/driver/led/NoLeds.h

```diff
@@ -0,0 +1,41 @@
+/* -*- mode: c++ -*-
+ * NoLeds -- Dummy LED hardware driver for Kaleidoscope
```

Comment:
For parity with the NoOp keyscanner, this should probably be something like NoOpLedDriver